src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,47 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
 
+    public static final String DATE_PATTERN = "dd.MM.yyyy";
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_PATTERN);
```

Comment:
Make it a constant

Reply:
I made constant there - public static final String DATE_PATTERN = "dd.MM.yyyy"  isn't it?
 after I use DATE_PATTERN